code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -20,6 +20,8 @@
 	high_threshold_cleared = "<span class='info'>Your vision functions passably once more.</span>"
 	low_threshold_cleared = "<span class='info'>Your vision is cleared of any ailment.</span>"
 
+	visual = TRUE
```

Comment:
You even did eyes??? Great stuff

Reply:
No, eyes haven't been reworked, but will be reworked in the future.